### PR TITLE
Add systemd file for 16.04

### PR DESCRIPTION
**Pull request by MattiasGees**

### CODE REVIEW

File: manifests/agent/php.pp [hunk 1]
```diff
@@ -107,4 +107,16 @@
     notify  => Service[$newrelic_php_service],
   }
 
+  if ($::lsbdistrelease == '16.04'0 {
```
Comment (iuriaranda) [inline manifests/agent/php.pp]: Typo `0` -> `)`

Comment (MattiasGees) [reply manifests/agent/php.pp]: Fixed

**Comment by simonrondelez** [comment]
👍

**Comment by iuriaranda** [comment]
:shipit:

**Comment by venturel** [comment]
👍

**Pull request merged**